couplings/fesom/env_fesom.py

```diff
@@ -15,7 +15,7 @@ def prepare_environment(config):
             "EXP_ID": config["general"]["command_line_config"]["expid"],
             "iter_coup_regrid_method_ice2oce": "INTERPOLATE",
             "fesom_use_icebergs": config["fesom"].get("use_icebergs", ""), 
-            "BASIN_FILE": config["fesom"].get("basin_file"),
+            #"BASIN_FILE": config["fesom"].get("basin_file"),
```

Comment:
Can this line be removed?

Reply:
Yes, this line can be removed. I made some more clean up